test/bin/encore.js

```diff
@@ -101,11 +101,12 @@ module.exports = Encore.getWebpackConfig();
             expect(parsedOutput).to.be.an('object');
             expect(parsedOutput.modules).to.be.an('array');
 
-            // We expect 3 modules there:
+            // We expect 4 modules there:
+            // - webpack/runtime/chunk loaded
```

Comment:
Was this related to the babel-loader bump? V `8.2.2` causes an extra module to be in here?

Reply:
webpack itself. They refactored their runtime in the 5.25 release